wren-tests/src/test/java/io/wren/testing/TPCH.java

```diff
@@ -0,0 +1,232 @@
+                    Column.column("r_name", WrenTypes.VARCHAR, null, false, "r_name"),
+                    Column.column("r_comment", WrenTypes.VARCHAR, null, false, "r_comment")),
+            "r_regionkey");
+
+    private static final Model PART = Model.model("part",
+            "select * from tablePrefix.part",
+            List.of(
+                    Column.column("p_partkey", WrenTypes.INTEGER, null, false, "p_partkey"),
+                    Column.column("p_name", WrenTypes.VARCHAR, null, false, "p_name"),
+                    Column.column("p_mfgr", WrenTypes.VARCHAR, null, false, "p_mfgr"),
+                    Column.column("p_brand", WrenTypes.VARCHAR, null, false, "p_brand"),
+                    Column.column("p_type", WrenTypes.VARCHAR, null, false, "p_type"),
+                    Column.column("p_size", WrenTypes.INTEGER, null, false, "p_size"),
+                    Column.column("p_container", WrenTypes.VARCHAR, null, false, "p_container"),
+                    Column.column("p_retailprice", WrenTypes.DOUBLE, null, false, "p_retailprice"),
+                    Column.column("p_comment", WrenTypes.VARCHAR, null, false, "p_comment")),
+            "p_partkey");
+
+    private static final Model SUPPLIER = Model.model("supplier",
+            "select * from tablePrefix.supplier",
+            List.of(
+                    Column.column("s_suppkey", WrenTypes.INTEGER, null, false, "s_suppkey"),
+                    Column.column("s_name", WrenTypes.VARCHAR, null, false, "s_name"),
+                    Column.column("s_address", WrenTypes.VARCHAR, null, false, "s_address"),
+                    Column.column("s_nationkey", WrenTypes.INTEGER, null, false, "s_nationkey"),
+                    Column.column("s_phone", WrenTypes.VARCHAR, null, false, "s_phone"),
+                    Column.column("s_acctbal", WrenTypes.DOUBLE, null, false, "s_acctbal"),
+                    Column.column("s_comment", WrenTypes.VARCHAR, null, false, "s_comment")),
+            "s_suppkey");
+
+    private static final Model PART_SUPP = Model.model("partsupp",
+            "select * from tablePrefix.partsupp",
+            List.of(
+                    Column.column("ps_partkey", WrenTypes.INTEGER, null, false, "ps_partkey"),
+                    Column.column("ps_suppkey", WrenTypes.INTEGER, null, false, "ps_suppkey"),
+                    Column.column("ps_availqty", WrenTypes.INTEGER, null, false, "ps_availqty"),
+                    Column.column("ps_supplycost", WrenTypes.DOUBLE, null, false, "ps_supplycost"),
+                    Column.column("ps_comment", WrenTypes.VARCHAR, null, false, "ps_comment"),
+                    Column.column("partkey_suppkey", WrenTypes.VARCHAR, null, true, "concat(ps_partkey, '-', ps_suppkey)")),
+            "partkey_suppkey");
+
+    public static final List<Model> MODELS = List.of(ORDERS, LINEITEM, CUSTOMER, NATION, REGION, PART, SUPPLIER, PART_SUPP);
+
+    private TPCH() {}
```

Comment:
We do many thing in `static` scope. Consider to make TPCH be a normal class and do initialization in ctor.